src/full/Agda/Interaction/Highlighting/HTML.hs

```diff
@@ -297,4 +298,4 @@ code asIs = mconcat . if asIs
         (++ "#" ++
          Network.URI.Encode.encode (show pos))
          -- Network.URI.Encode.encode (fromMaybe (show pos) aName)) -- Named links disabled
-        (Network.URI.Encode.encode $ modToFile m)
+        (Network.URI.Encode.encode $ modToFile m "html")
```

Comment:
This is what I'm not sure about.
PTAL @vlopezj

Reply:
I think it should be HTML, as even markdown documents will become HTML at some point. If someone has a use case for a different extension then a flag can be added.